src/Adapter/Product/Image/Uploader/ProductImageUploader.php

```diff
@@ -76,13 +77,13 @@ class ProductImageUploader extends AbstractImageUploader
     private $isLegacyImageMode;
 
     /**
-     * @var ProductImageRepository
+     * @var \PrestaShop\PrestaShop\Adapter\Product\Image\Repository\ProductImageRepository
      */
     private $productImageRepository;
 
     /**
      * @param UploadSizeConfigurationInterface $uploadSizeConfiguration
-     * @param ProductImagePathFactory $productImagePathFactory
+     * @param \PrestaShop\PrestaShop\Adapter\Product\Image\ProductImagePathFactory $productImagePathFactory
```

Comment:
why do you put the entire namespace ?

Reply:
I didn't it's PHPStorm which does this, when I refecto the classes I thought I fixed them all but I missed this one, thanks 😉